test/moonwall.config.json

```
@@ -8,15 +8,18 @@
             "name": "dev_tanssi",
             "envVars": ["DEBUG_COLORS=1"],
             "testFileDir": ["suites/dev-tanssi", "suites/common"],
+            "timeout": 120000,
+            "include": ["**/*test*"],
             "multiThreads": true,
-            "reporters": ["basic"],
+            "reporters": ["basic", "html", "json"],
+            "reportFile": { "json": "./tmp/testResults.json" },
             "foundation": {
                 "type": "dev",
                 "launchSpec": [
                     {
                         "name": "tanssi",
                         "binPath": "../target/release/tanssi-node",
-                        "options": ["--dev", "--sealing=manual", "--no-hardware-benchmarks"],
+                        "options": ["--dev", "--sealing=manual", "--no-hardware-benchmarks", "--tmp"],
```

`--dev` already implies `--tmp`:

```
      --dev
          Specify the development chain. This flag sets `--chain=dev`, `--force-authoring`, `--rpc-cors=all`, `--alice`, and `--tmp` flags, unless explicitly overridden
```
```suggestion
                        "options": ["--dev", "--sealing=manual", "--no-hardware-benchmarks"],
```